tests/playwright/src/utility/operations.ts

```diff
@@ -214,6 +214,26 @@ export async function deletePodmanMachine(page: Page, machineVisibleName: string
   }
 }
 
+export async function handleResetDefaultConnectionDialog(page: Page): Promise<void> {
+  const connectionDialog = page.getByRole('dialog', { name: 'Podman' });
+  const dialogMessage = connectionDialog.getByText(
+    new RegExp(
+      `.* Podman Machine .+ does not match default connection .+ Do you want to update the default connection?`,
+    ),
+  );
+  if ((await connectionDialog.isVisible()) && (await dialogMessage.isVisible())) {
+    const handleButton = connectionDialog.getByRole('button', { name: 'Yes' });
```

Comment:
Shouldn't checking for the existance/enablement of this button be sufficient? The button only exists when the popup is displayed so there is no point in checking at this time connectionDialog.isVisible() and dialogMessage.isVisible()